src/components/Drawer.js

```diff
@@ -8,7 +8,7 @@ const keycode = require('keycode');
 const PropTypes = require('prop-types');
 const React = require('react');
 const Velocity = require('velocity-animate');
-const { StyleRoot } = require('radium');
+import { css } from 'glamor';
```

Comment:
Was @whiteinge trying to remove glamor from the OS components? Where did we get on that? Is there an package we are trying to get to?

Reply:
I had the same question.

Reply:
I think that is the plan, but it hasn't been implemented yet.

Reply:
PR #900 makes the switch off Glamor. Looks like there's a new npm audit failure :roll_eyes: so I'll get that PR updated and then I think it's ready for prime-time.

That said, this PR looks fine and I don't see any reason the Glamor switch should hold this one up.